clang/test/SemaSYCL/Inputs/sycl.hpp

```diff
@@ -314,6 +314,16 @@ class handler {
     using NameT = typename get_kernel_name_t<KernelName, KernelType>::name;
     kernel_parallel_for_work_group<NameT>(kernelFunc, kh);
   }
+  template <typename KernelName = auto_name, typename KernelType, int Dims>
```

Comment:
I assume this matches existing headers?

Reply:
It does, but that change shouldn't be in there (i.e. not needed for the test I will add).  I will remove the change.